### ISSUE TITLE
Quick-Start guide leads to a broken page and errors on orchidServe

### ISSUE THREAD

**Issue by arturbosch** [orchidhq/OrchidStarter#9]
Hi there.

when following the quick start guide I get some errors and warnings on the console. The served site seems kinda broken too.

![2020-07-15-233507_1919x979_scrot](https://user-images.githubusercontent.com/20924106/87601225-f3091500-c6f4-11ea-827c-c90b80920d21.png)
![2020-07-15-233422_1231x681_scrot](https://user-images.githubusercontent.com/20924106/87601226-f3a1ab80-c6f4-11ea-8a55-2fa54ef76e20.png)

I've tested with both Gradle 5.6.2 and [6.5.1](https://github.com/orchidhq/OrchidStarter/pull/8) and Java 8 and 14.

I hope this kinda vague issue report helps.

**Comment by cjbrooks12** [comment]
Yeah, I'm working on getting better error messages here. Basically, there's an issue rendering the main portion of the screen, so nothing is being added there, but it doesn't report it correctly with included templates yet. I'll take a look and see what specifically is broken, thanks for the report.

**Comment by v-bilous** [comment]
@cjbrooks12 , @arturbosch hi guys, do u know any workarounds 4 this particular issue? 
I wanna upgrade orchid version 4 my site but facing exactly the same issue.

UPD: the issue is related to Orchid web server (local run), by some reason FileController.findPage takes https://orchid.run instead of https://localhost:8080

UPD2: Seems like the issue related to the logic introduced in 0.19.0: "dev server now dynamically renders indexed pages"
I was able to rid of the issue by setting the env variable `legacyFileServer=true`. Probably the issue isn't gonna happen on production since there are pre-rendered static pages only. 
UPD3: there is some conflicting functionality added for external pages
